### PR TITLE
Update code owners file

### PR DESCRIPTION
**Pull request by AlexGhiondea**
It seems like these entries are not correctly tracked.

Also fix a duplicate entry for Storage.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -594,3 +571,21 @@ sdk/trafficmanager/Microsoft.Azure.Management.TrafficManager/            @tmsupp
 
 # ServiceLabel: %Web Apps %Service Attention
 #/<NotInRepo>/            @AzureAppServiceCLI @antcp
+
+# ######## Management Plane ########
+
+/**/*Management*/                               @allenjzhang @m-nash @markcowl @YalinLi0312 @bquantump @nisha-bhatia
```
Comment (jsquire) [inline .github/CODEOWNERS:577]: This seems to match the pattern that you called out in your title, but the trailing `/` is still in place.   Was that intentional?

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -594,3 +571,21 @@ sdk/trafficmanager/Microsoft.Azure.Management.TrafficManager/            @tmsupp
...
+/**/Azure.ResourceManager*/                     @allenjzhang @m-nash @markcowl @YalinLi0312 @bquantump @nisha-bhatia
+
+# Reviewers to double check any API changes
+/sdk/**/api/                                    @KrzysztofCwalina @tg-msft
+
+# Update AutoRest C#
+/eng/CodeGeneration.targets                     @pakrym @ShivangiReja
+
+
```
Comment (jsquire) [inline .github/CODEOWNERS:586]: nit:  Extra blank line?

**Review by jsquire — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (AlexGhiondea) [reply .github/CODEOWNERS:580]: The issue is not with the pattern. It is with the fact that people/teams who don't have write permissions to the repo can't be added as owners in the file.

**Review by jsquire — APPROVED** [review]

**Pull request merged**